docs/en_us/internal/testing.rst

```diff
@@ -610,6 +616,35 @@ During acceptance test execution, Django log files are written to
 
 **Note**: The acceptance tests can *not* currently run in parallel.
 
+
+Testing internationalization with dummy translations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Any text you add to the platform should be internationalized. To generate
+translations for your new strings, run the following command.
+
+::
+
+    paver i18n_dummy
+
+This command generates dummy translations for each dummy language in the
+platform and puts the dummy strings in the appropriate language files.
+You can then preview the dummy languages on your local machine and also in
+your sandbox, if and when you create one.
+
+The dummy files that are generated during this process can be found in
+the following locations.
+
+::
+
+    conf/locale/en
+    conf/locale/eo
+    conf/locale/rtl
```

Comment:
Nope, they're in `lms/static/js/i18n/{LANG_CODE}` and `cms/static/js/i18n/{LANG_CODE}`.